### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by ViniciusSilva3**
Add Create user endpoint at POST /api/user.

The request body has following format:
```
{
    "cpf": "123456788",
    "password": "1234567",
    "cardNumber": "1234567890",
    "cardDate": "2020/01/02",
    "cardDigits": "555"
}
```

### CODE REVIEW

File: v2.0/ES/Constants/DatabaseConstants.cs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+namespace ES.Constants;
+
+public static class DbConstants
+{
+    public static string userFileName => "./Files/Usuarios.txt";
+}
```
Comment (zerbinidamata) [inline v2.0/ES/Constants/DatabaseConstants.cs]: Missing line in the end of the file

File: v2.0/ES/Db/Persister.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        switch (operation.Type)
+        {
+            case OperationType.WRITE:
+                File.WriteAllLines(operation.FileName, operation.Content);
+                return;
+            default:
+                throw new InvalidOperationException();
+        }
+    }
+}
```
Comment (zerbinidamata) [inline v2.0/ES/Db/Persister.cs]: Same here

File: v2.0/ES/Services/UserService.cs [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+        if (createUserOperation.HasValue)
+        {
+            _persister.ApplyChanges(createUserOperation.Value);
+            return Result.Ok();
+        }
+        
+        return Result.Fail("User already exists.");
+    }
+}
```
Comment (zerbinidamata) [inline v2.0/ES/Services/UserService.cs]: Same

**Review by zerbinidamata — CHANGES_REQUESTED** [review]
LGTM! Some files are missing an empty line at the end, some tools can't find the EOF without that empty line

**Comment by felipebayres** [comment]
👍

**Review by zerbinidamata — APPROVED** [review]

**Comment by zerbinidamata** [comment]
👍

**Pull request merged**